src/input.cpp

```diff
@@ -384,7 +384,7 @@ static bool input_mapping_is_match(const input_mapping_t &m) {
     const wcstring &str = m.seq;
 
     assert(str.size() > 0 && "zero-length input string passed to input_mapping_is_match!");
-    debug(4, L"trying to match mapping %ls", escape_string(m.seq.c_str(), ESCAPE_ALL).c_str());
+    debug(8, L"trying to match mapping %ls", escape_string(m.seq.c_str(), ESCAPE_ALL).c_str());
```

Comment:
Honestly: Just remove this one. There's not been a bug here for quite a while, and we wouldn't use this debug to catch it.